zeq/http/server.cpp

```diff
@@ -310,6 +310,18 @@ const servus::URI& Server::getURI() const
     return _impl->uri.toServusURI();
 }
 
+SocketDescriptor Server::getSocketDescriptor() const
```

Comment:
It's a commonly used pattern. You get an opaque handle to poll, but should not use it for something else. ZMQ provides it, X11 ConnectionNumber does, and many more.